SolveSATWithGrover/Tests.qs

```diff
@@ -249,35 +248,23 @@ namespace Quantum.Kata.GroversAlgorithm {
         AssertOperationsEqualReferenced(4,
             QubitArrayWrapperOperation(oracle(_, _, [[(1, false), (2, false)], [(0, true), (1, true)], [(1, false), (0, false)], [(2, true), (1, true)]]), _),
             QubitArrayWrapperOperation(Oracle_AlternatingBits_Reference, _));
-
-        // Standalone tests
-        for (i in 1..8) {
-            let (nVar, problem) = Generate_SAT_Instance(2);
-            Message($"Testing 2-SAT instance ({nVar}, {SATInstanceAsString(problem)})...");
-
-            AssertOracleImplementsFunction(nVar, oracle(_, _, problem), F_SAT(_, problem));
-
-            AssertOperationsEqualReferenced(nVar + 1,
-                QubitArrayWrapperOperation(oracle(_, _, problem), _),
-                QubitArrayWrapperOperation(Oracle_SAT_Reference(_, _, problem), _)
-            );
-        }
     }
 
     // ------------------------------------------------------
     @Test("QuantumSimulator")
     operation T16_Oracle_SAT_Test () : Unit {
         // General SAT oracle should be able to implement all 2SAT problems
-        Run2SATTests(Oracle_SAT);
+        RunCrossTests(Oracle_SAT);
 
-        // General SAT instances
-        for (i in 1..5) {
-            let (nVar, problem) = Generate_SAT_Instance(-1);
-            Message($"Testing k-SAT instance ({nVar}, {SATInstanceAsString(problem)})...");
+        // General SAT instances for 3..6 variables
 
-            AssertOracleImplementsFunction(nVar, Oracle_SAT(_, _, problem), F_SAT(_, problem));
+        for (i in 3..6) {
+            let problem = GenerateSatInstance(i,3,-1);
```

Comment:
The operation name mismatch was causing the build failure
```suggestion
            let problem = GenerateSATInstance(i, 3, -1);
```

Reply:
I'd also add some variance in the number of clauses - it's better to cover different scenarios
```suggestion
            let problem = GenerateSATInstance(nVar, nVar - 1, -1);
```

Reply:
Addressed in latest commit